src/content/reference/react/useContext.md

```diff
@@ -1288,9 +1288,9 @@ export const LevelContext = createContext(0);
 
 ---
 
-### Optimizing re-renders when passing objects and functions {/*optimizing-re-renders-when-passing-objects-and-functions*/}
+### Optimiser les rendus en passant des objets et des fonctions {/*optimizing-re-renders-when-passing-objects-and-functions*/}
```

Comment:
```suggestion
### Optimiser les rendus lorsqu'on passe des objets et des fonctions {/*optimizing-re-renders-when-passing-objects-and-functions*/}
```

Sinon in aurait l'impression que c'est le fait de passer ça qui optimise les rendus, alors qu'au contraire…